### PR TITLE
Test illustrations

### PR DESCRIPTION
**Pull request by mnaumanali94**

### CODE REVIEW

**Comment by lottamus** [comment]
Deploy preview for *stoplightio* ready!

Built with commit 975c57b0c9e5a92c3ea614a09c3703ca18ae5ac2

https://deploy-preview-1467--stoplightio.netlify.app

**Comment by paulatulis** [comment]
I really like this and I think that this is in the direction that Marc likes with the 'low-res' illustrations of the product. It's clean, clear and large. Nice work! 

Might try something different with the particles if we are getting rid of the dark background - probably a discussion that will change after we decide on adding a video/gif at the top.

**Comment by paulatulis** [comment]
Home Page: 

If we do low-res, it should be more low-res 
We should try high-res - in the style of Sketch

Keep two/three main sections - we have too many 
Secondary use cases should be less emphasized and underneath two main use cases 

Three main use cases:

- Collab - mocking under collaboration
- Visibility 
- Governance - style guides under governance

High res composition of the UI for each main use case

We can reuse icons for the smaller cards 

Represent practical showing of what we do - could be the video 

Are we keeping product images for use cases?
- We need a way to spice it up - follow sketch website with overlapping images

Secondary color: blue/purple

Update feature strip color if we leave color off of hero section for landing pages

Open Source Section in header

**Comment by paulatulis** [comment]
@mnaumanali94 
A few landing page items to take care of content-wise: 

/development
- Git Workflows icon needs a section for anchor link to go to 
- Icons and images update

/design
- Icons and images update

/mocking 
- Icons and images update

/documentation
- Multiple versions needs a section for anchor link to go to 
- Icons and images update

/visibility
- Icons and images update 

/governance
- Icons and images update

/enterprise
- Icons and images update

**Comment by paulatulis** [comment]
@mnaumanali94  do you have an svg for this? 

in collaboration.yaml it is `/images/mocking/mock.svg` but that doesn't exist in images folder

Screen Shot 2020-06-19 at 12 35 16 PM

**Comment by mnaumanali94** [comment]
Yes. I’ll push it in a bit. Missed this.

**Comment by ksatx** [comment]
Hey @mnaumanali94 and @paulatulis I just updated the paths for the use cases pages to be `/api-{{use case}}` and added the SEO content back onto `/design` `/documentation` `/mocking` per Scott's feedback in slack. I think I got all the hrefs updated appropriately, but we should run through links again before launch to be safe.
scott-seo-feeback

File: src/components/Hero/LandingHero.tsx [hunk 1]
```diff
@@ -0,0 +1,167 @@
...
+              </div>
+            )}
+
+            {list && (
+              <div className="pt-20">
+                <h2>{list.title}</h2>
+                <div className={cn('flex flex-col sm:flex-col mt-8 sm:mt-8  sm:w-100', {})}>
+                  {list?.items?.map((item, i) => (
+                    <div className="flex flex-row mb-10 sm:mx-10" key={i}>
+                      <Icon icon="arrow-right" size="lg" style={{ color: 'blue2' }} className="mt-1" />
```
Comment (paulatulis) [inline src/components/Hero/LandingHero.tsx:122]:
```suggestion
                      <Icon icon="arrow-right" size="lg" style={{ color: 'blue2' }}  />
```

**Pull request merged**